### ISSUE TITLE
Improve development workflow

### ISSUE THREAD

**Issue by vharsh** [lambainsaan/RemakingDave#14]
Hello mates, I am a really excited to share the opportunity of working with you. RemakingDave is about to half a week old now and with our love and support I can visualize a younger 'me' playing it. To enhance the throughput I suggest a few ideas of specifying some documents like

-  a CONTRIBUTING.md
-  pointers to absolute beginners(who have no python/pygame experience)
- a coding style-guide(like pep8)

Besides that let's follow something called [The Joel Test](https://www.joelonsoftware.com/2000/08/09/the-joel-test-12-steps-to-better-code/), aside by using git we can safely check the first criteria 😄 

Feel free to reach out, it's *our* project.

**Comment by lambainsaan** [comment]
I totally agree with you on this. We have to make it beginner friendly and the code must be more readable. I will work on most of these tasks tomorrow, I will keep you updated on the status of this issue. 

I am concerned about the structure of the project. Is it well structured, the files and stuff?
